news/2019-01-16-community-choice-2018.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Community Choice: 2018"
+date: 2019-01-16 04:00:00 +0000
+---
+
+It's time to take a look at 2018 in retrospect, and decide which 10 maps from each game mode will become the highly vaunted Community Favourites!
+
+[![](/wiki/shared/news/2019-01-16-community-choice-2018/header.jpg)](https://osu.ppy.sh/community/contests/74)
+
+If you're an older player, you might be more familiar with the Community Choice voting as the old "best of" contests held long ago around this time of year. Never fear, Community Choice is basically exactly the same thing, except with more style. We decided to change the name from "best of" to encourage a more rounded look at the voting and to have the result better reflect the kind of achievement it truly is.
+
+Here's how all this goes down:
+
+Every player gets **10 votes per game mode** to cast for their most adored **ranked** maps of 2018 **that they have played at least once**. Any maps you're eligible to vote on will *automagically* show up in the voting listing.
+
+Loved maps are currently not eligible.
```

Comment:
This seems kind of hanging out randomly in the middle of the article. Maybe start a new list of bullets?